readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -2830,35 +2829,37 @@ def test_get_active_versions(self):
         """Test the full response of
         ``/api/v2/version/?project__slug=pip&active=true``"""
         pip = Project.objects.get(slug='pip')
+        get(Version, project=pip, active=False, privacy_level=PUBLIC)
 
-        data = QueryDict('', mutable=True)
```

Comment:
This query dict was wrong, when using update() the second time it merged the values instead of replace them, e.g `?projec__slug=pip&project__slug=pip&active=true&active=false`, using a normal dictionary works better.